task.go

```diff
@@ -81,21 +83,38 @@ func RunTask(name string) error {
 		return &taskNotFoundError{name}
 	}
 
-	for _, d := range t.Deps {
-		if err := RunTask(d); err != nil {
-			return err
-		}
-	}
-
 	if !Force && isTaskUpToDate(t) {
```

Comment:
@andreynering  Why running dependencies for a task when the task itself would not run? That was at least my assumption, as you would create side-effects that possible could change what the task would generate itself. I create an issue for discussion and commit a change running deps first.